ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh

```diff
@@ -293,17 +293,29 @@ function run {
     echo "done to handle result"
     if [ "W${ret_value}W" == "W0W" ]; then
         echo "success"
-        exit 0
+    else
+        echo "fail"
     fi
-    echo "fail"
     # it ensure the the step after this step in test will be executed per https://docs.ci.openshift.org/docs/architecture/step-registry/#workflow
     # please refer to the junit result for case result, not depends on step result.
     if [ "W${FORCE_SUCCESS_EXIT}W" == "WnoW" ]; then
         echo "force success exit"
         exit 1
     fi
-    echo "normal exit"
-    exit 0
+
+    # summarize test results
+    echo "Summarizing test result..."
```

Comment:
@liangxia 
the following steps also need it?
openshift-extended-test-longduration
openshift-extended-test-disruptive
openshift-extended-logging-test

I saw the chain you add openshift-e2e-test-qe-report into includes the above step.

Reply:
Added.